share/functions/__fish_complete_lsusb.fish

```diff
@@ -1,3 +1,3 @@
 function __fish_complete_lsusb
-    lsusb | awk '{print $2 ":" $4}' | cut -c1-7
+    lsusb | awk -F[ :] '{ print $2 ":" $4 }'
```

Comment:
I'm talking about all the awk and sed stuff in general. Often it can be tricky to figure out if you're relying on non-standard behavior here.

Also, I think it's customary here to list Aho, Weinberger, Kernigan in that order - since that's where the acronym came from :smile:.